api/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.jboss.weld</groupId>
 		<artifactId>weld-parent</artifactId>
-		<version>31</version>
+		<version>36</version>
```

Comment:
Unrelated to this PR, it strikes me that the CDI spec's API is dependent on the impl.  I feel like it should be the other way around.